CHANGELOG.md

```diff
@@ -1,3 +1,151 @@
+# 108.0.0
+
+This major release includes several API additions to BottomSheet, Dialogs, and NavigationDrawer, along with a bug fix to Snackbar and improved Buttons documentation.
+
+## New deprecations
+
+MDCSemanticColorScheme's `init` was deprecated. Please use initWithDefaults: instead.
+
+## New features
+
+### Creating an `MDCAlertController` with attributed text
```

Comment:
It is clearer when using the same term consistently.  
Since you mentioned "Dialogs" in the top description blurb, then let's use "Dialogs" here and throughout (instead of the "MDCAlertController").

Reply:
Done